task3/fields.js

```diff
@@ -0,0 +1,30 @@
+let form = document.forms.test;
+let enterfield = form.elements.enter;
+let copyfield = form.elements.copy;
```

Comment:
Why `let` variable here? Where you will change the value?
Use camel case for variables: `enterField` and etc.